src/app/public/modules/select-field/types/select-field.ts

```diff
@@ -1,6 +1,13 @@
 export interface SkySelectField {
+
+  category?: string;
+
+  description?: string;
```

Comment:
This property was missing, but is used by List view checklist to render a description.